src/fprime/util/cookiecutter_wrapper.py

```diff
@@ -178,9 +178,35 @@ def new_deployment(build: Build, parsed_args: "argparse.Namespace"):
             + "/../cookiecutter_templates/cookiecutter-fprime-deployment"
         )
         print("[INFO] Cookiecutter: using builtin template for new deployment")
+
+    # Determine the include path prefix based on the current directory
+    proj_root = Path(build.get_settings("project_root", None)).resolve()
+    cwd = Path.cwd()
+    extra_context = {}
+
+    # If we're in a subdirectory of the project root, set the include_path_prefix
+    if proj_root != cwd and proj_root in cwd.parents:
+        # Get the relative path from project root to current directory
+        rel_path = cwd.relative_to(proj_root)
+
+        # Ask for confirmation before setting the include path prefix
+        if confirm(
```

Comment:
Any reason to ask for confirmation here? I'd be tempted to say we should just go ahead and do it anyways and include a `[INFO]` log message. But you have more vision into the use case than I do

Reply:
Good point, I usually try to keep old behavior together with new behavior, but in this scenario there is not much use for the old behavior I guess 🤷‍♂️.